code/modules/mapping/space_management/traits.dm

```diff
@@ -49,6 +49,15 @@
 				. += S.z_value
 				break
 
+/// Get a list of all z which have all of the specified traits
+/datum/controller/subsystem/mapping/proc/levels_by_all_traits(z, list/traits)
+	. = list()
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-
You shouldn't be using `.` like this anymore